src/rules/tags-check.js

```diff
@@ -39,80 +39,80 @@ function assign(target) {
     return target;
 }
 
-HTMLHint.addRule({
+export default {
     id: 'tags-check',
     description: 'Checks html tags.',
     init: function (parser, reporter, options) {
-        var self = this;
+      var self = this;
 
-        if (typeof options !== 'boolean') {
-            assign(tagsTypings, options);
-        }
+      if (typeof options !== 'boolean') {
+          assign(tagsTypings, options);
+      }
 
-        parser.addListener('tagstart', function (event) {
-            var attrs = event.attrs;
-            var col = event.col + event.tagName.length + 1;
+      parser.addListener('tagstart', function (event) {
```

Comment:
we should convert all of this functions to arrow-functions
but we should do this for the whole codebase and use eslint to find all of this occurrences

Reply:
That's the plan right after we merge all old PRs :)